lib/index.js

```diff
@@ -135,40 +131,46 @@ class DeploymentBucketPlugin {
   }
 
   async applyDeploymentBucket() {
-    try {
-      if (await this.bucketExists(this.deploymentBucket.name)) {
-        this.serverless.cli.log(`Using deployment bucket '${this.deploymentBucket.name}'`)
-      } else {
-        this.serverless.cli.log(`Creating deployment bucket '${this.deploymentBucket.name}'...`)
+    if (this.deploymentBucket.name) {
```

Comment:
I don't like wrapping the entire block around like this.
It would be better to short circuit out of the function.
Invert the conditional (`if (!this.deploymentBucket.name) {`) and exit out of the function instead.  Reduce nesting.